internal/integration_test/engine/threads_test.go

```diff
@@ -57,6 +57,10 @@ func TestThreadsInterpreter(t *testing.T) {
 }
 
 func incrementGuardedByMutex(t *testing.T, r wazero.Runtime) {
+	P := 8               // max count of goroutines
+	if testing.Short() { // Adjust down if `-test.short`
```

Comment:
Sorry I should have done this originally but forgot, it is the same pattern as the other hammer tests. I notice CI is run with the `-short` flag